.github/workflows/release.yml

```diff
@@ -114,7 +114,7 @@ jobs:
             stack-2-${{ runner.os }}-ghc-${{ env.ghc_version }}-${{ hashFiles('stack.yaml') }}
             stack-2-${{ runner.os }}-ghc-${{ env.ghc_version }}
 
-      - uses: haskell-actions/setup@v2
+      - uses: haskell-actions/setup@v2.6.1
```

Comment:
Not actually necessary... I am watching https://github.com/haskell-actions/setup/pull/66 to upgrade this action as soon as the version based on node-20 is available.

Reply:
Reverted